@stellar/anchor-tests/src/helpers/console.ts

```diff
@@ -76,15 +76,6 @@ async function printColoredTextTestRun(testRun: TestRun, verbose: boolean) {
       console.groupEnd(); // resource links group
     }
   }
-  if (verbose && testRun.result.failure && testRun.result.failure.links) {
-    console.log(c.bold("Relevant Links:\n"));
-    console.group(); // failure links group
-    for (const linkLabel in testRun.result.failure.links) {
-      console.log(`${linkLabel}: ${testRun.result.failure.links[linkLabel]}`);
-    }
-    console.groupEnd(); // failure links group
-    console.log();
-  }
```

Comment:
just curious why this is being removed in this PR. With my limited context, it looks unrelated to SEP-38 tests

Reply:
Its just duplicate logic, the section above already outputs the links. I noticed when running via the CLI that the links were outputted twice when using verbose.